project/newEvaluation.html

```diff
@@ -1,472 +1,489 @@
-                <option value="0">0</option>
-                <option value="1">1</option>
-                <option value="2">2</option>
-                <option value="3">3</option>
-              </select>
-            </li>
-            <li>
-              <label for="frame">Frameworks</label>
-              <select name="frame" id="brow">
-                <option value="0">0</option>
-                <option value="1">1</option>
-                <option value="2">2</option>
-                <option value="3">3</option>
-              </select>
-            </li>
-        </ul>
-      </fieldset>
+          <fieldset>
+            <legend>Javascript</legend>
+            <ul>
+              <li>
+                  <label for="data">Data types and variables</label>
+                    <select name="data" id="data">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="object">Object manipulation</label>
+                    <select name="object" id="object">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="dom">DOM manipulation</label>
+                    <select name="dom" id="dom">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                </li>
+                <li>
+                  <label for="funct">Functions</label>
+                    <select name="funct" id="funct">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="temp">Templating</label>
+                    <select name="temp" id="temp">
+                      <option value="0">0</option>
+                      <option value="1">1</option>
+                      <option value="2">2</option>
+                      <option value="3">3</option>
+                    </select>
+                  <label for="eventh">Event Handling</label>
+                  <select name="eventh" id="eventh">
+                    <option value="0">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    </select>
+                </li>
+                <li>
+                  <label for="prot">Prototype & OOP</label>
```

Comment:
I have to write its entity name which I cannot write here because it will transform it into simple & :))